tests/unit/test_routes/test_resource_update.py

```diff
@@ -256,29 +254,56 @@ def test_update_resource(
     assert_correct_response(response, 404)
 
 
-def test_update_votes_authorization_header(
-        module_client, module_db, fake_auth_from_oc, fake_algolia_save):
```

Comment:
I'm going to need this test to be added back in. I'm pretty sure it was deleted due to the cherry-picking situation, not an intentional delete. In any case, please just add it back in somewhere in the file because this is a very important test